README.md

```diff
@@ -936,6 +936,23 @@ yarn start options rename-code \
   --post
 ```
 
+### analyze
+
+Analyze option codes and generates a csv report if codes:
+
+-   Are duplicated in other optionSets
+-   Include any special characters (only underscore is allowed)
+-   length is greater than 230 (you can change this value --code-length [VALUE])
+-   have spaces or commas
+
+```shell
+yarn start options analyze-codes \
```

Comment:
Ok for now, if we end up checking more codes, we can rename it for something more general like "validate"